### PR TITLE
New olas schema

### PR DESCRIPTION
**Pull request by ShubhamChndrvnshi**
This PR aims to change the type of contentUrl from bytes32 to string

### CODE REVIEW

**Review by KohliSuraj — DISMISSED** [review]
Looks good to me

**Comment by KohliSuraj** [comment]
@ShubhamChndrvnshi see an error on the RPC the tests are using, will have to update that.

**Comment by ShubhamChndrvnshi** [comment]
> @ShubhamChndrvnshi seeing an error on the RPC the tests are using, will have to update that.

The test should be able to run if we update the rpc url. On local they were working with fork-url.

**Review by RojhatToptamus — DISMISSED** [review]

File: src/abis/AuthorStake_ABI.json
[the remark this answers is not in the document]
Comment (ShubhamChndrvnshi) [reply src/abis/AuthorStake_ABI.json:1]: generated new abis with the script generate_contract_abis.js. Also pushed the attestations in recent commit.

[the remark this answers is not in the document]
Comment (KohliSuraj) [reply src/abis/AuthorStake_ABI.json:1]: @RojhatToptamus Good point, had missed this.

**Review by KohliSuraj — APPROVED** [review]

**Pull request merged**